lib/Getopt/Std.pm

```diff
@@ -146,7 +147,7 @@ sub getopt (;$$) {
     }
     unless (ref $hash) { 
 	local $Exporter::ExportLevel = 1;
-	import Getopt::Std;
+	Getopt::Std->import;
```

Comment:
I'm not very sure, might be due to `Exporter`, but it's "argument" is a bareword, so looks like a reverse method syntax.

Reply:
Yeah this is indirect syntax.